toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexServiceImpl.kt

```diff
@@ -274,6 +279,33 @@ class CratesLocalIndexServiceImpl
         private const val CRATES_INDEX_VERSION: Int = 0
 
         private val LOG: Logger = logger<CratesLocalIndexServiceImpl>()
+
+        private val _TEST_INSTANCE: CratesLocalIndexService by lazy {
```

Comment:
`Object or top-level property name '_TEST_INSTANCE' should not start with an underscore`
Let's use `TEST_INSTANCE` instead. Looks like there is no conflict with `getTestInstance` because of uppercase